uber/config.py

```diff
@@ -412,3 +412,9 @@ def _is_intstr(s):
     c.VOLUNTEER_CHECKLIST = [url for step, url in _items]
 
 stripe.api_key = c.STRIPE_SECRET_KEY
+
```

Comment:
Agreed. I did a search for `c.DOOR_PAYMENT_METHODS` and `c.DOOR_PAYMENT_VARS` and we aren't in fact using them, but I was thinking it would be much nicer to edit the enums _before_ they were processed.